components/CandidateTestView/CandidateTestView.tsx

```diff
@@ -0,0 +1,31 @@
+import scss from "../CandidateTestView/candidateTestView.module.scss";
+import {limitedAutoComplete} from "../../Models/TextEditorOptions";
+import React from "react";
+import {CandidateTestModel} from "../../Models/CandidateTestModel";
+import TextEditorContainer, {getEditorProps} from "./TextEditor/TextEditorContainer";
+import InstructionsBlock from "./Instructions/Instructions";
+
+const CandidateTestView = (testModel: CandidateTestModel): JSX.Element => (
+    <div>
+        <h1 className={scss.testTitle}>Test {testModel.number}: {testModel.title}</h1>
+        <div className={scss.testPage}>
+            <section className={scss.testInstructions}>
+                <h2 className={scss.instructionsHeader}>{testModel.title}</h2>
+                <div id={"testInstructions"} className={scss.instructions}>
+                    {InstructionsBlock(testModel)}
+                </div>
+            </section>
+            <section className={scss.textEditor}>
+                <TextEditorContainer containerClassName="textEditor"
+                                     theme="dark"
+                                     height="50vh"
+                                     width="50vw"
+                                     language="javascript"
+                                     defaultText={getEditorProps(testModel).defaultText}
+                                     options={limitedAutoComplete}
+                />
```

Comment:
this also still looks out of date.

This is the current 'TextEditorContainer' - it has a different set of props. https://github.com/techswitch-learners/testswitch-frontend-candidate/blob/master/components/TextEditor/TextEditorContainer.tsx